music/src/core/visualizer.ts

```diff
@@ -163,22 +162,20 @@ export abstract class BaseVisualizer {
 
   protected scrollIntoViewIfNeeded(
       scrollIntoView: boolean, activeNotePosition: number) {
-    if (scrollIntoView && this.parentElement) {
-      // See if we need to scroll the container.
+    if (scrollIntoView) {
       const containerWidth = this.parentElement.getBoundingClientRect().width;
-      if (activeNotePosition >
-          (this.parentElement.scrollLeft + containerWidth)) {
-        this.parentElement.scrollLeft = activeNotePosition - 20;
-      }
+      this.parentElement.scrollLeft = activeNotePosition - containerWidth * 0.5;
```

Comment:
Let's not change this, please. You can overwrite this behaviour if you want in a custom visualizer, but I find the "keep the active note in the middle" a worse experience (it moves a lot too much and it makes it harder to follow what's happening).

Reply:
Sure! Will do. 

Please notice current pagination scrolling has a cummulative bug due to extra width of the outer DIV padding (noticeable in longer scores). Not very urgent stuff, so I'll try to fix it in further contributions.

Reply:
@notwaldorf Done!